src/to-string.cpp

```diff
@@ -453,6 +456,7 @@ const char * get_string( rs2_format value )
     CASE( Y411 )
     CASE( Y16I )
     CASE( M420 )
+    CASE( OCCUP )
```

Comment:
Full name might be clearer?

Reply:
full name leads to bad UI in our viewer - this is the reason I have shorten it

Reply:
So we can do something else,
use full name as enum all over, shorten the name for the viewer here.
Try using the full name in the header and here replace this line with
```
case RS2_FORMAT_OCCUPANCY: return "OCCUP";
```

Reply:
ok